client-admin/src/util/with-auth0-ready.js

```diff
@@ -0,0 +1,74 @@
+// Higher-Order Component to ensure Auth0 is ready before executing API calls
```

Comment:
Logging here as well so we don't miss anything - this kind of check should not be necessary. When the app mounts, we should instantiate auth0, and then proceed with showing the rest of the components that depend on it